api/core/v1alpha1/authorization_types.go

```diff
@@ -166,7 +166,7 @@ type ClusterAdminStatus struct {
 // +kubebuilder:subresource:status
 
 // ClusterAdmin is the Schema for the cluster admin API
-// +kubebuilder:resource:shortName=clas
+// +kubebuilder:resource:shortName=class
```

Comment:
I am not sure if it is `clas` by intension. This would change a CRD. I would not touch it.